[2022-10-11 11:51:10,391] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps manual__2022-10-11T18:51:07.213928+00:00 [queued]>
[2022-10-11 11:51:10,397] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps manual__2022-10-11T18:51:07.213928+00:00 [queued]>
[2022-10-11 11:51:10,398] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-11 11:51:10,398] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-11 11:51:10,398] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-11 11:51:10,412] {taskinstance.py:1377} INFO - Executing <Task(DbtDepsOperator): dbt_deps> on 2022-10-11 18:51:07.213928+00:00
[2022-10-11 11:51:10,415] {standard_task_runner.py:52} INFO - Started process 217324 to run task
[2022-10-11 11:51:10,420] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dbt', 'dbt_deps', 'manual__2022-10-11T18:51:07.213928+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/dbt_airflow.py', '--cfg-path', '/tmp/tmpdyh22s2k', '--error-file', '/tmp/tmp8hijwy1_']
[2022-10-11 11:51:10,421] {standard_task_runner.py:80} INFO - Job 54: Subtask dbt_deps
[2022-10-11 11:51:10,472] {task_command.py:370} INFO - Running <TaskInstance: dbt.dbt_deps manual__2022-10-11T18:51:07.213928+00:00 [running]> on host ubuntu
[2022-10-11 11:51:10,525] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dbt
AIRFLOW_CTX_TASK_ID=dbt_deps
AIRFLOW_CTX_EXECUTION_DATE=2022-10-11T18:51:07.213928+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-11T18:51:07.213928+00:00
[2022-10-11 11:51:10,526] {dbt_hook.py:117} INFO - dbt deps
[2022-10-11 11:51:10,531] {dbt_hook.py:126} INFO - Output:
[2022-10-11 11:51:13,757] {dbt_hook.py:130} INFO - 18:51:13  Running with dbt=1.1.0
[2022-10-11 11:51:13,774] {dbt_hook.py:130} INFO - 18:51:13  Warning: No packages were found in packages.yml
[2022-10-11 11:51:15,545] {dbt_hook.py:132} INFO - Command exited with return code 0
[2022-10-11 11:51:15,556] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dbt, task_id=dbt_deps, execution_date=20221011T185107, start_date=20221011T185110, end_date=20221011T185115
[2022-10-11 11:51:15,601] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-11 11:51:15,625] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
